### ISSUE TITLE
Move first/last to Layer from List

### ISSUE THREAD

**Issue by azaroth42** [IIIF/iiif.io#553]
After engaging with the W3C ActivityStreams folks about paging through collections, they came to a new pattern which is quite appealing.  The only significant difference with the current search draft is that first and last are relationships from the Layer to the AnnotationList (in our speak), rather than from AnnotationList to AnnotationList.

Semantically, this seems more accurate.  It is the first page of the Layer, not of some random non-first AnnotationList.  We always require the Layer for paged results, and the first/last is consistent.  I propose that we follow suit and move these to Layer.

Concretely:

```
{
  "@context":"http://iiif.io/api/search/0/context.json",
  "@id":"http://example.org/service/manifest/search?q=bird&page=1",
  "@type":"sc:AnnotationList",
  "within": {
    "@type": "sc:Layer",
    "total": 125,
    "ignored": ["box"]
  },
  "next": "http://example.org/service/identifier/search?q=bird&page=2",
  "last": "http://example.org/service/identifier/search?q=bird&page=13",
  "startIndex": 0,
```

Would become:

```
{
  "@context": "http://iiif.io/api/search/0/context.json",
  "@id": "http://example.org/service/manifest/search?q=bird&page=1",
  "@type": "sc:AnnotationList",
  "within": {
    "@type": "sc:Layer",
    "total": 125,
    "first": "http://example.org/service/manifest/search?q=bird&page=1",
    "last": "http://example.org/service/identifier/search?q=bird&page=13",
    "ignored": ["box"]
  },
  "next": "http://example.org/service/identifier/search?q=bird&page=2",
  "startIndex": 0,
```

**Comment by jpstroop** [comment]
I agree, this seems more semantically accurate. It's a breaking change, no?

Sent via mobile. Please excuse typos, brevity, etc.

**Comment by azaroth42** [comment]
It would be if search had been published, but it's still draft.

**Comment by jpstroop** [comment]
Ah, duh.

**Comment by tomcrane** [comment]
It is a breaking change for me! But only a tiny one and trivially altered. I've just finished switching to the draft model for autocomplete and search, but it's not live yet in Wellcome manifests (Ed needs to add to UV). I'll make this change now, I agree it makes sense.

**Comment by tomcrane** [comment]
I've made this change (first and last move to layer, prev and next stay on annotation list).

"prev" looks ugly - any reason why this can't be "previous"? The other terms are not abbreviated.

**Comment by azaroth42** [comment]
We could map to `previous` in the context.

The previous iteration of AS, before my comments towards alignment:
    http://www.w3.org/TR/activitystreams-core/#collections

That said, in 3.0 we should try and just use external contexts directly
rather than re-naming. By then both AS and Annotations will be through to
TR.  So my preference is to leave it as prev to not have to re-re-name it.

**Comment by tomcrane** [comment]
Yes, fair enough!

**Comment by azaroth42** [comment]
Done in latest search branch (v 0.9.1)

Branch: http://search.iiif.io/api/search/0.9/

+1s?

**Comment by jpstroop** [comment]
:+1:

**Comment by azaroth42** [comment]
Closed in #650
